ta/sims_keepalive/ta_sims_keepalive.c

```diff
@@ -0,0 +1 @@
+../sims/ta_sims.c
```

Comment:
No symbolic link please, they can cause all sorts of troubles. Use `#include "../sims/ta_sims.c"` instead.

Reply:
Wasn't aware about the problem. Eventually, this has been fixed.